x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/relevance_tuning_form/relevance_tuning_form.tsx

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useActions, useValues } from 'kea';
+
+import {
+  EuiPageHeader,
+  EuiPageHeaderSection,
+  EuiTitle,
+  EuiFieldSearch,
+  EuiSpacer,
+  EuiAccordion,
+  EuiPanel,
+} from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+
+import { FIELD_FILTER_CUTOFF } from '../constants';
+import { RelevanceTuningLogic } from '../relevance_tuning_logic';
+
+import { RelevanceTuningItem } from './relevance_tuning_item';
+import { RelevanceTuningItemContent } from './relevance_tuning_item_content';
+
+import './relevance_tuning_form.scss';
+
+export const RelevanceTuningForm: React.FC = () => {
+  const {
+    filterInputValue,
+    schemaFields,
+    filteredSchemaFields,
+    schema,
+    searchSettings,
+  } = useValues(RelevanceTuningLogic);
+  const { setFilterValue } = useActions(RelevanceTuningLogic);
+
+  return (
+    <section className="relevanceTuningForm">
+      <form>
+        {/* TODO SchemaConflictCallout */}
+
+        <EuiPageHeader>
+          <EuiPageHeaderSection>
+            <EuiTitle size="m">
+              <h2>
+                {i18n.translate(
+                  'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.manageFields.title',
+                  {
+                    defaultMessage: 'Manage fields',
+                  }
+                )}
+              </h2>
+            </EuiTitle>
+          </EuiPageHeaderSection>
+        </EuiPageHeader>
+        {schemaFields.length > FIELD_FILTER_CUTOFF && (
+          <>
+            <EuiSpacer />
+            <EuiFieldSearch
+              value={filterInputValue}
+              onChange={(e) => setFilterValue(e.target.value)}
+              placeholder={i18n.translate(
+                'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.manageFields.filterPlaceholder',
+                {
+                  defaultMessage: 'Filter {schemaFieldsLength} fields...',
+                  values: {
+                    schemaFieldsLength: schemaFields.length,
+                  },
+                }
+              )}
+              fullWidth
+            />
+          </>
```

Comment:
I don't think we need the extra EuiSpacer here, it looks a bit odd visually. We can likely just make this `EuiFieldSearch`
```suggestion
          <EuiFieldSearch
            value={filterInputValue}
            onChange={(e) => setFilterValue(e.target.value)}
            placeholder={i18n.translate(
              'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.manageFields.filterPlaceholder',
              {
                defaultMessage: 'Filter {schemaFieldsLength} fields...',
                values: {
                  schemaFieldsLength: schemaFields.length,
                },
              }
            )}
            fullWidth
          />
```

Reply:
5374281

Reply:
https://github.com/elastic/kibana/pull/90621/commits/537428191b428f7259cdf5732f1b62068bd20859